plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotIntegrationConnectorSmokeTest.java

```diff
@@ -1548,19 +1548,6 @@ public void testLimitPushdown()
                 .isNotFullyPushedDown(LimitNode.class);
     }
 
-    @Test
-    public void testPredicatePushdown()
-    {
-        assertThat(query("SELECT timestamp_col FROM " + ALL_TYPES_TABLE + " WHERE timestamp_col < TIMESTAMP '1971-01-01 00:00:00.000'"))
```

Comment:
Is there any test cases verifying timestamp values except for `1971-01-01 00:00:00.000`? I have concern about the current test coverage.

Reply:
Added.